### ISSUE TITLE
Fix H1 on OD-radio episode for a11y AT

### ISSUE THREAD

**Issue by RichardPK** [bbc/simorgh#6579]
**Describe the bug**
On the NVDA screen reader - the h1 on OD-radio episodes is 2 presses of the down arrow key, this should be one. 

It is 1 press for the brand title and 1 press for the date, it should be just 1 that does both

**To Reproduce**
Steps to reproduce the behaviour:

(Using the NVDA screen reader)
1. Go to 'https://www.test.bbc.com/indonesia/bbc_indonesian_radio/w172xh267fpn19l'
2. Navigate to the H1
3. Note that it is 2 presses of the down arrow key

**Approach**
To fix please replace the display block on the brand title span with display inline-block and width 100% P2

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**
Flagged during a11y swarm of OD radio page.
